js/src/forum/components/DiscussionListItem.js

```diff
@@ -98,22 +97,21 @@ export default class DiscussionListItem extends Component {
 
         <div className={'DiscussionListItem-content Slidable-content' + (isUnread ? ' unread' : '') + (isRead ? ' read' : '')}>
           <a
-            href={user ? app.route.user(user) : '#'}
+            route={user ? app.route.user(user) : '#'}
             className="DiscussionListItem-author"
             title={extractText(
               app.translator.trans('core.forum.discussion_list.started_text', { user: user, ago: humanTime(discussion.createdAt()) })
             )}
-            config={function (element) {
-              $(element).tooltip({ placement: 'right' });
-              m.route.apply(this, arguments);
+            oncreate={function (vnode) {
+              $(vnode.dom).tooltip({ placement: 'right' });
```

Comment:
This was previously called on `onupdate` as well. Did that simply have no effect?

Reply:
Well, once we've created the tooltip, it remains on the DOM element until the DOM element is removed, right? (in which case oncreate would run again if its readdded). There's no need to re-update that if we're not changing the text (which we're not).

Reply:
Thanks. I wasn't familiar with how the tooltip JS works. :+1: